bundle/src/main/java/com/adobe/acs/commons/oak/impl/EnsureOakIndexManagerImpl.java

```diff
@@ -74,10 +74,12 @@ public class EnsureOakIndexManagerImpl extends AnnotatedStandardMBean implements
     private static final Logger log = LoggerFactory.getLogger(EnsureOakIndexManagerImpl.class);
 
     //@formatter:off
-    private static final String[] DEFAULT_ADDITIONAL_IGNORE_PROPERTIES = new String[]{};
-    private String[] additionalIgnoreProperties = DEFAULT_ADDITIONAL_IGNORE_PROPERTIES;
     public static final String PROP_ADDITIONAL_IGNORE_PROPERTIES = "properties.ignore";
 
+    /**
+     * The OSGi configuration is only used by the EnsureOakIndex OSGi services and would have ideally been a separate OSGi service,
+     * but we can't migrate this easily without losing the PID for existing implementations.
```

Comment:
Have you considered renaming this class and separate the config keeping the old PID?

Reply:
This makes this code comment obsolete then, right?